app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -77,6 +74,14 @@ class ConfigurationHeuristic(private val heuristicConfigurationData: HeuristicCo
       new HeuristicResultDetails(
```

Comment:
Here is the scenario. If I have a job that fails both dynamic allocation check and kyroserializer check you will take the max of severity and show that.  Let's say that is dynamic allocation check. Once that is fixed, they will rerun the job and find it failing the kyroserializer check now. I am asking to make it obvious to the user they have multiple issues to fix within this one heuristic.